docs/create_allocator_service.md

```diff
@@ -0,0 +1,335 @@
+# Quickstart Create an Allocator Service
```

Comment:
I'm feeling that this isn't actually a quickstart - it should be more of a guide - "How to Create A Fleet Allocation via the Kubernetes API"

Because this is actually what we are showing here, the service aspect is more of an implementation detail in my mind - what we really want to show is "how do you create a fleet allocation through the API".

WDYT?

Reply:
Yes, I had the same question regarding is it a guide or a quickstart several weeks ago.  Looking at other documents in this repo in a general way, Quickstarts seem to leave the reader with something tangible, whereas Guides seem to be more descriptions of things that exist.

If showing code that creates a fleet allocation is the goal, then all you would need is something like the go snippet in access_api.md.  Lines 159 - 182 of main.go in this pr could suffice.  While that would be more helpful than no code snippet at all, I think it would leave some readers wondering what to do with the snippet.

The service aspect is definitely an implementation detail, but one that seems necessary if the user is to easily see their code snippet in action.  It moves us from a review of how one could theoretically perform a fleet allocation programmatically to actually doing so.  It may also generate some useful discussion about calling the API.

I think having a working example of how to call the API is useful.  If a reader finds this to be too complicated, they are free to consider alternative solutions, having gained a bit of knowledge of one way to use the API.  Reading through the slack channel, it's a question that comes up and having some docs that address the idea seems helpful to me.